gravitee-am-repository/gravitee-am-repository-jdbc/src/main/java/io/gravitee/am/repository/jdbc/management/api/spring/SpringEventRepository.java

```diff
@@ -30,4 +30,7 @@
 public interface SpringEventRepository extends RxJava3CrudRepository<JdbcEvent, String> {
     @Query("select * from events e where e.updated_at >= :from and e.updated_at <= :to")
     Flowable<JdbcEvent> findByTimeFrame(@Param("from") LocalDateTime from, @Param("to") LocalDateTime to);
+
+    @Query("select * from events e where e.updated_at >= :from and e.updated_at <= :to and e.data_plane_id = :dataPlaneId")
```

Comment:
issue: IMO we should adapt this query to search by dataplaneID or by null dataplaneID. Some event sent by the mAPI may not have those informations (org level sync event for example)